example/data/collections/s3_MOD09GQ_006_full_ingest/s3_MOD09GQ_006.json

```diff
@@ -14,8 +14,7 @@
     "orca": {
       "excludedFileExtensions": [".met"]
     },
-    "s3MultipartChunksizeMb": 16,
-    "uniquifyGranuleId": false
```

Comment:
This change will result in roughly 8 tests (the ones that aren't overriding this value)  moving from non-unqified testing to uniqified granule testing - are we certain that's not creating testing gaps around full ingest behavior where we aren't uniqifying granules?

Reply:
Ah, I see, you're just removing an explicit default.    Got it.